tests/test_task_api.py

```diff
@@ -17,22 +15,26 @@ async def test_task_get_list(db, client, task_factory, MockCurrentUser):
         assert data[2]["id"] == t2.id
 
 
-async def test_task_create(db, client, MockCurrentUser):
+async def test_collection_api(client, dummy_task_package, MockCurrentUser):
     """
-    GIVEN a CreateTask object
-    WHEN it is fed to the `POST task` endpoint
-    THEN a new task is correctly created
+    GIVEN a package in a format that `pip` understands
+    WHEN the api to collect tasks from that package is called
+    THEN an environment is created, the package is installed and the task
+         collected
     """
-    task = TaskCreate(
-        name="mytask",
-        command="cmd",
-        source="/source",
-        input_type="Any",
-        output_type="Any",
+    PREFIX = "/api/v1/task"
+
+    task_collection = dict(
+        collection_type="pypi", package=dummy_task_package.as_posix()
```

Comment:
Is `collection_type` meant to be `pypi` or `pip`?

Reply:
(by the way, this argument of `create_task_pip` is currently never used.. maybe it was some leftover from past code or a placeholder for future features)

Reply:
correct, will remove